.github/workflows/terraform-batch.yml

```diff
@@ -15,74 +15,74 @@ jobs:
+        git pull origin main --rebase
 
-      - name: Setup Terraform
-        if: steps.count.outputs.count != '0'
-        id: tf
-        uses: bcgov/sso-requests-actions/actions/setup-terraform@v0.39.0
-        with:
-          context: ./terraform
-          tf-version: ${{ env.TF_VERSION }}
-          tf-s3-bucket: "xgr00q-prod-keycloak"
-          tf-s3-bucket-key: "keycloak/silver"
-          tf-s3-access-key: ${{ secrets.TF_S3_ACCESS_KEY }}
-          tf-s3-secret-key: ${{ secrets.TF_S3_SECRET_KEY }}
-          tf-s3-role-arn: ${{ secrets.TF_S3_ROLE_ARN }}
-          kc-dev-url: ${{ env.KEYCLOAK_DEV_URL }}
-          kc-test-url: ${{ env.KEYCLOAK_TEST_URL }}
-          kc-prod-url: ${{ env.KEYCLOAK_PROD_URL }}
-          kc-dev-secret: ${{ secrets.KEYCLOAK_DEV_CLIENT_SECRET }}
-          kc-test-secret: ${{ secrets.KEYCLOAK_TEST_CLIENT_SECRET }}
-          kc-prod-secret: ${{ secrets.KEYCLOAK_PROD_CLIENT_SECRET }}
-          apply: true
-        continue-on-error: true
+    - name: Setup Terraform
+      if: steps.count.outputs.count != '0'
+      id: tf
+      uses: bcgov/sso-requests-actions/actions/setup-terraform@v0.39.0
+      with:
+        context: ./terraform
+        tf-version: ${{ env.TF_VERSION }}
+        tf-s3-bucket: xgr00q-prod-keycloak
+        tf-s3-bucket-key: keycloak/silver
+        tf-s3-access-key: ${{ secrets.TF_S3_ACCESS_KEY }}
+        tf-s3-secret-key: ${{ secrets.TF_S3_SECRET_KEY }}
+        tf-s3-role-arn: ${{ secrets.TF_S3_ROLE_ARN }}
+        kc-dev-url: ${{ env.KEYCLOAK_DEV_URL }}
+        kc-test-url: ${{ env.KEYCLOAK_TEST_URL }}
+        kc-prod-url: ${{ env.KEYCLOAK_PROD_URL }}
+        kc-dev-secret: ${{ secrets.KEYCLOAK_DEV_CLIENT_SECRET }}
+        kc-test-secret: ${{ secrets.KEYCLOAK_TEST_CLIENT_SECRET }}
+        kc-prod-secret: ${{ secrets.KEYCLOAK_PROD_CLIENT_SECRET }}
+        apply: true
+      continue-on-error: true
 
-      - name: Setup NPM Packages
-        if: steps.count.outputs.count != '0'
-        uses: bcgov/sso-requests-actions/actions/setup-yarn@v0.39.0
+    - name: Setup NPM Packages
+      if: steps.count.outputs.count != '0'
+      uses: bcgov/sso-requests-actions/actions/setup-yarn@v0.39.0
 
-      - name: Update Requests Status
-        if: steps.count.outputs.count != '0'
-        uses: fjogeleit/http-request-action@master
-        with:
-          url: "${{ env.API_URL }}/batch/items"
-          method: "PUT"
-          data: '{"ids": ${{ steps.requests.outputs.response }}, "success": "${{ steps.tf.outputs.apply != ''failure'' }}"}'
-          customHeaders: '{"Authorization": "${{ secrets.GH_SECRET }}"}'
-          timeout: "60000"
+    - name: Update Requests Status
+      if: steps.count.outputs.count != '0'
+      uses: fjogeleit/http-request-action@master
+      with:
+        url: ${{ env.API_URL }}/batch/items
+        method: PUT
+        data: "{\"ids\": ${{ steps.requests.outputs.response }}, \"success\": \"${{ steps.tf.outputs.apply != 'failure' }}\"}"
```

Comment:
if this was formatted by yaml formatter then it must be tested before going ahead with this change